---
layout: post
title: "B6. Genome-wide association study for variants associated with reading abilities"
header-img: "img/banner.jpg"
category: abstracts
platform: "neuro"
subtitle: "Kaitlyn Price, Karen G Wigg, Virginia Misener, Kirsten Blokland, Elizabeth Kerr, Sharon Guger, Maureen W Lovett, Cathy L Barr"
tags: poster neuro
session_id: B6 Poster_Session_2
visible: true
---
Poster Session 2 - B6

**<sup>1,2,3</sup>Kaitlyn Price**, <sup>1</sup>Karen G Wigg, <sup>1</sup>Virginia Misener, <sup>2</sup>Kirsten Blokland, <sup>2,3</sup>Elizabeth Kerr, <sup>2,3</sup>Sharon Guger, <sup>2,3</sup>Maureen W Lovett, <sup>1,2,3</sup>Cathy L Barr

__¹ Krembil Research Institute, University Health Network, Toronto, ON, Canada
2 The Hospital for Sick Children, Toronto, ON, Canada
3 Dept of Physiology, University of Toronto, Toronto, ON, Canada__

Reading disability (RD) is the most common neurodevelopmental disorder affecting children in North America. Although RD is known to be a complex genetic trait, associated genes largely remain unknown and unreplicated. We sought to identify genetic variants associated with reading by performing genome wide association (GWA) analyses on 5253 individuals from two well characterized samples. The primary sample is a family-based RD selected sample from Toronto and the secondary is a population-based unselected sample named the Philadelphia Neurodevelopmental Cohort (PNC). Both samples were previously measured for word reading ability and genotyped. We performed quality control analysis, imputation, association analyses and meta-analysis. We hypothesized the majority of our top associated SNPs would be in novel genes and that we would find additional genes associated with RD related comorbidities in our selected sample. Top associated SNPs from Toronto and PNC samples were found in genes (NRCAM, BIN1, KANSL2, and CCNT1) that are associated with other neurodevelopmental disorders and/or transcriptional processes. These SNPs are located in enhancer regions. Enhancers modulate transcription in a cell-type specific manner. Based on these findings, we further hypothesize that RD associated SNPs may alter enhancer function leading to altered gene expression. We are testing our top associated SNPs’ effect on enhancer function in neural precursor cells using a luciferase reporter assay. We also found a variant (rs2071267) in BIN1 which is predicted to be a splice site. We are testing for alternatively spliced isoforms associated with this variant in our RD samples. This will contribute to a better functional understanding of variants associated with RD as well as associated neurodevelopmental disorders. 